web/views/webclient_templates.xml

```diff
@@ -195,7 +195,7 @@
                 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
                 <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
 
-                <title t-esc="title or 'Odoo'"/>
+                <title t-esc="title or 'M-TIBA'"/>
```

Comment:
This should remain as `Odoo, powered by m-tiba` being that the product itself is Odoo.